proc_scripts/align_slice.py

```diff
@@ -35,22 +36,23 @@ def correlation_align(s,avg,convwidth=0,axis='t2',fl=None):
     R = pi*convwidth # Lorentzian FWHM formula -- R = 1/T2, FWHM = R/pi
     # ALSO convolve both
     check = avg * s * exp(-2*R*abs(s.fromaxis(axis)))
+    print(check)
     assert s.get_ft_prop(axis,['start','freq']) is not None, "should be FT'd first, so the startpoint is set"
-    check.ft(axis, pad=2**14)
-    check.reorder(axis,first=False)
-    forplot = abs(check)[axis:(-50,50)]
-    myline = forplot.C.argmax(axis)
+    #check.ft(axis, pad=2**14)
+    #check.reorder(axis,first=False)
+    #forplot = abs(check)[axis:(-50,50)]
```

Comment:
why this range on axis? what dictates this?

Reply:
the typical linewidth on our instrument